apps/condo/domains/marketplace/access/Invoice.js

```diff
@@ -31,36 +37,28 @@ async function canReadInvoices ({ authentication: { item: user }, context }) {
             return false
         }
 
-        if (residents.length > 50) {
+        if (residents.length > RESIDENTS_COUNT_WARNING_THRESHOLD) {
             logger.warn({
-                msg: `user ${user.id} has ${residents.length} residents`,
-                reqId: context.req.id,
-                userId: user.id,
-            })
-        }
-
-        const serviceConsumers = await find('ServiceConsumer', {
-            deletedAt: null,
-            resident: { id_in: residents.map(({ id }) => id) },
-        })
-
-        if (serviceConsumers.length > 50) {
-            logger.warn({
-                msg: `user ${user.id} has ${serviceConsumers.length} service consumers`,
+                msg: `user has ${RESIDENTS_COUNT_WARNING_THRESHOLD}+ residents`,
```

Comment:
Then now you can delete this comment and receive only number of residents, instead of their data